### DISCUSSION TITLE
statictoc template output

### DISCUSSION THREAD

**Discussion by osisoft-mbishop** [dotnet/docfx#8547] [Q&A]
Hi @yufeih 

I have a dumb question. I'm trying to build out a docfx project using the statictoc template, but each time I build, docfx outputs the default template. I've tried both updating docfx.json `templates` node to `"templates": "statictoc"` and running `docfx docfx.json -t statictoc`, but to no avail. It sounds simple enough in the docs, but I can't make it happen.

I'm running docfx 2.63.1.

Here's what my docfx.json build node looks like:

```json
"build": {
		...
		"templates": "statictoc",
		...
	}
```

How can I get docfx to output the statictoc template? Any hints would be appreciated.

**Comment by yufeih** [comment]
Does `"templates": ["default", "statictoc"]` work?

**Comment by osisoft-mbishop** [comment]
@yufeih Unfortunately it does not. I also tried “templates”: [“default”, “modern”] just to test, and that’s not working either.

**Comment by osisoft-mbishop** [comment]
Bumping this to get some attention. Any suggestions @yufeih?
